controllers/topics.controller.js

```diff
@@ -0,0 +1,12 @@
+const { request, response } = require("express");
```

Comment:
It looks like VScode has auto imported these! They should be removed as you don't use them in your code.